genetic_algorithms/problems/traveling_salesman_problem/benchmarks/tsp_for_visualization

```diff
@@ -0,0 +1,39 @@
+0
+11003 42102
```

Comment:
Z ciekawości: czemu akurat takie punkty?

Reply:
Czy to jest jakoś związane ze współrzędnymi w pygamie?
Jeśli tak, to myślę, że lepiej by było zostawić jakieś czytelne liczby, a potem w pygamie to skalować.

Reply:
Chciałem użyć jakiegoś dużego testu i taki znalazłem w necie, który miał odpowiedź. Raczej nie ma związku ze współrzędnymi w pygamie, bo i tak musiałem je skalować. 
Faktycznie, czytelne liczby byłyby lepsze.
Tutaj ten konkretny test:
![image](https://user-images.githubusercontent.com/84449847/134592477-deed9b0e-1354-4484-ad45-bb30a1fb1cc6.png)

Reply:
Dobra. 
To w takim razie fajny pomysł, i myślę, że warto go zostawić.